### PR TITLE
KOGITO-1518: Handle null response from server

### PR DESCRIPTION
**Pull request by Sara4994**

### CODE REVIEW

**Review by AjayJagan — APPROVED** [review]

File: packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx [hunk 1]
```diff
@@ -97,6 +99,89 @@ const DomainExplorerColumnPicker: React.FC<IOwnProps> = ({
...
+         tempObj[val[0]] = null;
+         valueObj = {...valueObj , ...tempObj}
+         return valueObj;  
+      }
+    }
+  }
+
+  const validateResponse = (obj) => {
+    let contentObj= {}
+    // tslint:disable-next-line: forin
```
Comment (cristianonicolai) [inline packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx]: @Sara4994 can we avoid disabling the tslint rules?

File: packages/management-console/src/components/Organisms/DomainExplorerTable/DomainExplorerTable.tsx [hunk 2]
```diff
@@ -119,9 +119,9 @@ const DomainExplorerTable = ({ columnFilters, tableLoading, displayTable }) => {
       }
     };
     const tempKeys = [];
-    let tempValue = [];
+    const tempValue = [];
     iter(object);
-    tempValue = tempValue.filter(value => value !== null)
+    // tempValue = tempValue.filter(value => value !== null)
```
Comment (cristianonicolai) [inline packages/management-console/src/components/Organisms/DomainExplorerTable/DomainExplorerTable.tsx]: please remove coment out code.

File: packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx (hunk 1 above)
Comment (Sara4994) [reply packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx]: done

File: packages/management-console/src/components/Organisms/DomainExplorerTable/DomainExplorerTable.tsx (hunk 2 above)
Comment (Sara4994) [reply packages/management-console/src/components/Organisms/DomainExplorerTable/DomainExplorerTable.tsx]: @cristianonicolai done

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**